build.sbt

```diff
@@ -196,6 +196,7 @@ lazy val shared = (project in file("modules/shared"))
       Libraries.circeGenericExtras,
       Libraries.circeParser,
       Libraries.circeRefined,
+      Libraries.circeShapes,
```

Comment:
is it still needed? I think it was a test but with failure :D

Reply:
No, it can be removed 👍🏼